src/Microsoft.AspNetCore.Razor.Design/Internal/AssemblyTagHelperDescriptorResolver.cs

```diff
@@ -61,6 +52,47 @@ public IEnumerable<TagHelperDescriptor> Resolve(string assemblyName, ErrorSink e
                         DesignResources.InvalidProtocolValue,
                         typeof(TagHelperDescriptor).FullName, ProtocolVersion));
             }
+
+            if (!_loadedAssembly)
```

Comment:
Yes. The `Resolve` method is called every time you resolve an assembly. https://github.com/aspnet/RazorTooling/blob/dev/src/Microsoft.AspNetCore.Razor.Design/ResolveTagHelpersRunCommand.cs#L42